package.json

```diff
@@ -11,8 +11,8 @@
     "build:ssr": "npm run build && npm run webpack:server",
     "build:prerender": "npm run build:ssr && npm run generate:prerender",
     "build:prerender-ghpages": "npm run build:ghpages && npm run webpack:server && npm run generate:prerender",
-    "build:app": "ng build --prod",
-    "build:server": "ng build --prod --app 1 --output-hashing=false",
+    "build:app": "ng build --prod --project=ngx-drag-to-select",
```

Comment:
How about we call that only `app` and `server`?

Reply:
:+1: I think it was that way, but then I ran `ng update` which renamed it.